src/common/fixtures/category.ts

```diff
@@ -1,16 +1,15 @@
-import { Category } from '../../../src/module/category/application/domain/category.entity';
-export const category1: Category = {
```

Comment:
Create a type based on the real Category, so every time Category changes it makes you change the fixture. 
You can use typescript Partials or Exclude.  Or you can use mapper functions.

Reply:
Done